src/main.ts

```diff
@@ -6,7 +6,8 @@ import store from '@/store/store';
 import vuetify from '@/plugins/vuetify'; // path to vuetify export
 import 'roboto-fontface/css/roboto/roboto-fontface.css';
 //import "@mdi/font/css/materialdesignicons.css";
-import 'material-design-icons-iconfont/dist/material-design-icons.css';
+//import 'material-design-icons-iconfont/dist/material-design-icons.css';
```

Comment:
Can we remove these if they are not needed

Reply:
Agree